### PR TITLE
check-bugs: Set run to unstable when commands crash

### PR DESCRIPTION
**Pull request by thegreyd**
Right now there are times when [we silently fail](https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fcheck-bugs/684/console) - fix those cases 
Set the Jenkins run to Unstable if any of the commands fail

Test run
https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/sidsharm-aos-cd-jobs/job/build%252Fcheck-bugs/11/console

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/check_bugs.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __init__(self, runtime: Runtime, channel: str, versions: list, pre_releases:
         self.applicable_versions = []
         self.blockers = {}
         self.regressions = {}
-        self.slack_client = self.initialize_slack_client(runtime, channel)
+        self.slack_client = None if not channel else self.initialize_slack_client(runtime, channel)
```
Comment (thegreyd) [inline pyartcd/pyartcd/pipelines/check_bugs.py:30]: Don't force slack initialization when running locally

File: jobs/build/check-bugs/Jenkinsfile [hunk 2]
```diff
@@ -63,8 +63,8 @@ node {
                 def out = sh(script: cmd.join(' '), returnStdout: true).trim()
                 echo out
 
-                if (out.contains('failed with')) {
-                    currentBuild.result = "FAILURE"
+                if (out.contains('Command failed:')) {
```
Comment (vfreex) [inline jobs/build/check-bugs/Jenkinsfile:66]: `logging.info/warning` could print logs to stderr.

**Review by vfreex — CHANGES_REQUESTED** [review]

File: jobs/build/check-bugs/Jenkinsfile (hunk 2 above)
Comment (thegreyd) [reply jobs/build/check-bugs/Jenkinsfile]: I removed this, and now checking the return code of the script

**Review by vfreex — APPROVED** [review]
/lgtm

**Pull request merged**